handlers/handlers.go

```diff
@@ -102,7 +103,11 @@ func (d *CatalystAPIHandlersCollection) UploadVOD() httprouter.Handle {
 			errors.WriteHTTPInternalServerError(w, "Cannot validate payload", err)
 			return
 		} else if !result.Valid() {
-			errors.WriteHTTPBadRequest(w, "Invalid request payload", nil)
+			var errString string
+			for i, desc := range result.Errors() {
+				errString += fmt.Sprintf("%d - %s, ", i, desc)
+			}
+			errors.WriteHTTPBadRequest(w, "Invalid request payload: "+strings.TrimSuffix(errString, ", "), nil)
```

Comment:
Can you use [WriteHTTPBadBodySchema()](https://github.com/livepeer/catalyst-api/blob/4d1416eda13e869a7ed7089c5373f8a2c31e9ca6/errors/errors.go#L44)? The string builder code looks nicer.